### ISSUE TITLE
Fix download ESPIn repository example

### ISSUE THREAD

**Issue by mdpiper** [csdms/espin#28]
In the "Download the ESPIn repository" section of the [last](https://github.com/csdms/espin/blob/main/lessons/shell/getting-things.md) shell lesson, we ask participants to download the ESPIn repository as a zip archive with the `wget` command. The problem is that while `wget` is typically installed on Linux, it's not on macOS or Windows. This example should be fixed to work on all platforms.

*Expected behavior:* Download repo as zip archive
*Actual behavior:* Error message is thrown because `wget` isn't available

**Comment by mdpiper** [comment]
The most likely solution is to use `curl` instead of `wget`. Note, however, that the URL used in the example performs a redirect that `wget` handles but `curl` doesn't. Here's the correct way to do this with `curl`:
```
$ curl https://codeload.github.com/csdms/espin/zip/main -o main.zip
```

**Comment by mdpiper** [comment]
Fixed in #32.
